README.md

```diff
@@ -1 +1,105 @@
-# northstar
\ No newline at end of file
+# 🌟 North Star 
+
+# Meta
+
+> If your starting-point is unknown, and your end-point and intermediate stages are woven together out of unknown material, there may be coherence, but knowledge is completely out of the question.
+> 
+> — Plato, The Republic
+
+This document is an informal description of IPVM's high-level goals. It forms the basis for reasoning about the project from first principles. It expected to only rarely change.
+
+This is meant to provide the basis for reasoning from first principles, but it is written in prose and not first-order logic. It "merely" provides a basis for aligning the people working on the project. Principles often contain some limited internal tension: use your best judgment and discuss with your peers when ambiguities inevitably arise.
+
+This text should be kept short, to the point, and as unambiguous as possible. Define any project-specific jargon, avoid editorializing, and limit subtle language. Use diagrams and imagery where appropriate.
+
+# Raison d'Être
+
+> Had the [Web] been proprietary, and in my total control, it would probably not have taken off. You can’t propose that something be a universal space and at the same time keep control of it.
+>
+> — Tim Berners-Lee, [FAQ][TBL FAQ]
+
+IPVM intends to do nothing less than connect all of the world's users and services. It can be thought of as "the HTTP of compute": open, interoperable, and everywhere. Following the model of the Web being the network of linked documents, IPVM is the network of linked computation. It allows anyone to permissionlessly tie into the network without prenegotiation.
+
+TCP/IP and HTTP were successful for a number of reasons. Networking was always possible, but doing this from scratch for every application is infeasible. Hypertext provided a single mechanism for many applications to interconnect. Application agnosticism is a major pillar of [such an approach][W3F Principles]: anything can run on this substrate, and it's easy to join. IPVM works equally well with traditional Cloud-based services as it does local-first compute and trustless architectures. To be successful, IPVM must have a path towards becoming substantially better than a pure Cloud architecture. It does not stand in opposition to the Cloud, but rather extends and contains it.
+
```

Comment:
not sure where, but should we be explicit in support of browser as well as native? It's most def alluded too, but maybe more explicit (again somewhere?)?

Reply:
Yeah, I went back and forth on this, but also kind of wanted to leave it open as "wherever you want to run things". We don't call out that it can run on IoT, but we totally could if someone had a strong use case. The emphasis on the Cloud here is that Cloud computing is the status quo.

We totally _can_ add stuff about the browser if you still feel strongly about it though 👍

Reply:
Added to the goals section